src/PhpParser/Visitor/MutationCollectorVisitor.php

```diff
@@ -45,9 +45,6 @@
  */
 final class MutationCollectorVisitor extends NodeVisitorAbstract
 {
-    /**
-     * @var iterable<Mutation>[]
-     */
```

Comment:
have you understood why the fixer removes it?

Reply:
I think it interpreted it as `mixed[]` and resulted to a "superfluous phpdoc", I changed to to `array<...>` and it seems to do the trick